From b31f6c630cf1881ea2c61e055ef7fdd484c93609 Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <Michal@Hrusecky.net>
Date: Sat, 24 Nov 2018 00:45:00 +0100
Subject: [PATCH] python3: bump version to 3.6.6

There is problem with compilation with OpenSSL 1.1.0f on our build
servers. There seems to be some fixes for OpenSSL 1.1.0 later revisions
in Python 3.6.6 so let's test new version.
---
 lang/python/python3-version.mk | 2 +-
 lang/python/python3/Makefile   | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/lang/python/python3-version.mk b/lang/python/python3-version.mk
index dd6c947ac..48c67e4f7 100644
--- a/lang/python/python3-version.mk
+++ b/lang/python/python3-version.mk
@@ -8,7 +8,7 @@
 # Note: keep in sync with setuptools & pip
 PYTHON3_VERSION_MAJOR:=3
 PYTHON3_VERSION_MINOR:=6
-PYTHON3_VERSION_MICRO:=5
+PYTHON3_VERSION_MICRO:=6
 
 PYTHON3_VERSION:=$(PYTHON3_VERSION_MAJOR).$(PYTHON3_VERSION_MINOR)
 
diff --git a/lang/python/python3/Makefile b/lang/python/python3/Makefile
index 52d054f8d..d21fe9ca2 100644
--- a/lang/python/python3/Makefile
+++ b/lang/python/python3/Makefile
@@ -19,7 +19,7 @@ PKG_VERSION:=$(PYTHON_VERSION).$(PYTHON_VERSION_MICRO)
 
 PKG_SOURCE:=Python-$(PKG_VERSION).tar.xz
 PKG_SOURCE_URL:=https://www.python.org/ftp/python/$(PKG_VERSION)
-PKG_HASH:=f434053ba1b5c8a5cc597e966ead3c5143012af827fd3f0697d21450bb8d87a6
+PKG_HASH:=d79bc15d456e73a3173a2938f18a17e5149c850ebdedf84a78067f501ee6e16f
 
 PKG_LICENSE:=PSF
 PKG_LICENSE_FILES:=LICENSE Modules/_ctypes/libffi_msvc/LICENSE Modules/_ctypes/darwin/LICENSE Modules/_ctypes/libffi/LICENSE Modules/_ctypes/libffi_osx/LICENSE Tools/pybench/LICENSE
-- 
2.19.1

